Multisig/Logic/Models/AppSettings.swift

```diff
@@ -39,6 +39,18 @@ extension AppSettings {
     @AppSetting(\.displayMode)
     static var displayMode: Int32
 
+    @AppSetting(\.fiatCode)
+    private static var fiatCode: String?
```

Comment:
what is the difference between fiat code and selected fiat code? I see there's a default value, but maybe we could extend the property wrapper to accept a default value and not have 2 properties for the same thing?

Reply:
I couldn't updated it now, it seems this will need refactor for were we used it